### DISCUSSION TITLE
Student Dev pack approved; Not received the mail with access information yet

### DISCUSSION THREAD

**Discussion by DaideepyaV** [community/community#129051] [GitHub Education]
### Hi everyone,

My student dev pack shows to be approved, but I still have not received any mail with further information to access the benefits of the student dev pack.
![image](https://github.com/community/community/assets/172979481/7ff4761a-0dd7-4405-b6e1-f9f93bcf6746)

**Comment by davevad93** [comment]
Hi @DaideepyaV ,

You've been approved but you're benefits will be available soon, when they become available the state of your application will change to:

`Congrats! Your request for @DaideepyaV was approved and your benefits are now available`

It may take a while, from a few days to a few weeks.

**Reply by DaideepyaV** [reply]
I see. Thank you for the response. Do you think sending a request again would help?

**Reply by davevad93** [reply]
You're welcome @DaideepyaV , no, sending a new request wouldn't help because you've been already approved and your application is being processed by `GitHub Education`, so you have to wait till your benefits become available. The waiting time may vary depending on different factors, such as the volume of applications, it usually takes only a few days but there have been cases where it has taken a few weeks, so be patient! 👍

**Reply by mishrayush15** [reply]
Buddy, did you got the confirmation mail or maybe the approval yet ??, my application was also submitted and approved on July 16th and didn't get any kind of mail.

**Reply by DaideepyaV** [reply]
> You're welcome @DaideepyaV , no, sending a new request wouldn't help because you've been already approved and your application is being processed by `GitHub Education`, so you have to wait till your benefits become available. The waiting time may vary depending on different factors, such as the volume of applications, it usually takes only a few days but there have been cases where it has taken a few weeks, so be patient! 👍

Gotcha!

**Reply by DaideepyaV** [reply]
> Buddy, did you got the confirmation mail or maybe the approval yet ??, my application was also submitted and approved on July 16th and didn't get any kind of mail.

I'm waiting for the confirmation mail as well. Github says that it can take 5days max, so let's wait for 2-3 more days

**Reply by shivang0070** [reply]
hey @davevad93  ,

I am also waiting for the approval ,  haven't received any mail with further information to access the benefits of the student dev pack.
My student dev pack shows to be approved.
![image](https://github.com/community/community/assets/173198620/185f38bb-2ad4-406e-86a7-df13693abcfb)
![image](https://github.com/community/community/assets/173198620/135f2c15-dda7-4f30-8f29-1d80c09aa795)

**Reply by davevad93** [reply]
Hey @shivang0070 , 

I don't know why but GitHub lately isn't sending the emails notifying about the status of the request., e.g. see this discussion #128894 

Anyway it says that "`Your benefits will be available soon`" which means you still have to wait, your benefits will be available when the status changes to "`Congrats! Your request for @shivang0070 was approved and your benefits are now available`".

**Comment by kanduru-abhiram** [comment]
Hey @DaideepyaV, As per the image you posted I can see that your request is approved by Github Campus but please wait till the Approved button shaped image becomes violet, once turned violet, you'll have the access to all the benefits. Please check below image for reference. Accept this solution if useful.

![image](https://github.com/community/community/assets/85011280/257cc5f8-ad0f-4783-a9de-9d3f874f71d3)
